aliases/available/fuck.aliases.bash

```diff
@@ -1,3 +1,4 @@
-# https://github.com/nvbn/thefuck
-alias fuck='eval $(thefuck $(fc -ln -1))'
-alias please='fuck'
+alias please='sudo $(fc -ln -1)'
+alias plz=please
+alias fuck='sudo'
+alias fucking=fuck
```

Comment:
When I made my first comment, I was focused on trying to resolve the relationship between thefuck's "figure out how to fix it!" behavior and a more simplistic "just run the last command with sudo" behavior.

It did not occur to me that you meant for `fuck` to _just_ be an alias for `sudo` and not actually try to re-run the previous command.

I think of `fuck` as being a more-angry `please` - In the same way that the `thefuck` command intends it.
```
$ chown -R <new_owner> ./
# failed

$ fuck
# sudo chown -R <new_owner> ./
# success!
```

It also did not occur to me that `fucking` is not actually a good alias name for "just run the last command with sudo", but is good as a simple alias for `sudo`

```
$ chown -R <new_owner> ./
# failed

$ fucking chown -R <new_owner> ./
# success!
```

Additionally, with the aliases file being named `fuck.aliases.bash` AND with the idea of this being a more simplistic version of `thefuck`, I think `fuck` needs to be the primary alias here, with secondary aliases similar to the `thefuck` alias file.

So with all of the above, I think the file should probably look like:
```
alias fuck='sudo $(fc -ln -1)'
alias please=fuck
alias plz=please
alias fucking=sudo
```

Reply:
Agreed, this looks a lot better.